everyvoice/wizard/prompts.py

```diff
@@ -56,12 +65,17 @@ def get_response_from_menu_prompt(
         show_search_hint=search,
         status_bar_style=("fg_gray", "bg_black"),
     )
-    index = menu.show()
+    selection = menu.show()
     sys.stdout.write("\033[K")
-    if index is None or return_indices:
-        return index
+    if multi:
+        if selection is None:
+            return ()
+        elif return_indices:
+            return selection
+        else:
+            return [choices[i] for i in selection]
     else:
-        if isinstance(index, tuple):
-            return [choices[i] for i in index]
+        if return_indices:
+            return selection
         else:
-            return choices[index]
+            return choices[selection]
```

Comment:
Just for my understanding, when `multi` you can select nothing but otherwise you have to select something?

Reply:
Exactly.